scanner/src/main/kotlin/ScanResultsStorage.kt

```diff
@@ -144,20 +149,28 @@ abstract class ScanResultsStorage {
                 maxPoolSize = LocalScanner.NUM_STORAGE_THREADS + 3
             )
 
+            log.info {
```

Comment:
Hmm. I wonder whether we should move all this logging centrally to `createStorage()` instead.

Reply:
I prefer it this way to keep `createStorage()` concise. Esp. because the logging can also be conditional like in `createFileBasedStorage()`.